scss/_patterns_lists.scss

```diff
@@ -260,6 +260,12 @@ $spv-list-item--inner: null;
     }
 
     h#{$i}.p-stepped-list__title {
+      padding-left: $bullet-width-mobile + $sph-inner;
+
+      @media (min-width: $breakpoint-heading-threshold) {
+        padding-left: $bullet-width + $sph-inner;
+      }
+
       &::before {
         flex-shrink: 0;
```

Comment:
I guess we can remove that as well now that `flex` is gone from parent.

Reply:
true!

Reply:
@bartaz updated